docs/Running-Flink-on-Tachyon.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: global
+title: Running Apache Flink on Tachyon
+---
+
+This guide describes how to get Tachyon running with [Apache Flink](http://flink.apache.org/), so that you can easily work with files stored in Tachyon.
+
+# Prerequisites
+
+The prerequisite for this part is that you have
+[Java](https://github.com/amplab/tachyon/wiki/Java-setup/). We also assume that you have set up
+Tachyon and Flink in accordance to these guides [Local Mode](Running-Tachyon-Locally.html) or
+[Cluster Mode](Running-Tachyon-on-a-Cluster.html). 
+
+Plese find the guides for setting up Flink on the [Apache Flink website](http://flink.apache.org/).
+
+
+# Configuration
+
+Apache Flink allows to use Tachyon through a generic file system wrapper for Hadoop file systems. Therefore, the configuration of Tachyon is done mostly in Hadoop configuration files.
+
+#### Set property in `core-site.xml`
+
+If you have a Hadoop setup next to the Flink installation, add the following property to the `core-site.xml` configuration file:
```

Comment:
Thank you for reviewing my text Henry!

So in my understanding this depends on Hadoop (or Flink) and not on Tachyon.

The reason why there is no need to specify that configuration property for Hadoop 2 is that they are using the [ServiceLoader](http://docs.oracle.com/javase/6/docs/api/java/util/ServiceLoader.html) from Java. Thats a mechanism which is using some files in the `META-INF` directory in jars to detect available services.
So once the TachyonFS jar is in HDFS's classpath, it will (through the META-INF) file detect Tachyon as a available filesystem and load it on demand.

Hadoop1 was basically getting the classname for filesystems by getting the value for "fs.<scheme>.impl". 
Currently, Flink is handling the loading of the Hadoop FileSystem classes just using the configuration, not through the ServiceLoader.
Thats the reason why Flink users have to put the value for all Hadoop versions.